orchestrator/careplancontributor/config.go

```diff
@@ -14,12 +14,13 @@ func DefaultConfig() Config {
 }
 
 type Config struct {
-	CarePlanService   CarePlanServiceConfig `koanf:"careplanservice"`
-	FrontendConfig    FrontendConfig        `koanf:"frontend"`
-	AppLaunch         applaunch.Config      `koanf:"applaunch"`
-	FHIR              coolfhir.ClientConfig `koanf:"fhir"`
-	Enabled           bool                  `koanf:"enabled"`
-	StaticBearerToken string
+	CarePlanService           CarePlanServiceConfig `koanf:"careplanservice"`
+	FrontendConfig            FrontendConfig        `koanf:"frontend"`
+	AppLaunch                 applaunch.Config      `koanf:"applaunch"`
+	FHIR                      coolfhir.ClientConfig `koanf:"fhir"`
+	Enabled                   bool                  `koanf:"enabled"`
+	ZorginzageEndpointEnabled bool                  `koanf:"zorginzageendpointenabled"`
```

Comment:
can you add a unit test for `DefaultConfig()` that verifies it's disabled by default? Since it's important for security.

Reply:
Good callout, will do alongside the change to use an English term